src/log-stats.c

```diff
@@ -89,17 +89,16 @@ static int LogStatsLogger(ThreadVars *tv, void *thread_data, const StatsTable *s
     int days = in_hours / 24;
 
     MemBufferWriteString(aft->buffer, "----------------------------------------------"
-            "--------------------------------------\n");
+                                      "-----------------------------------------------------\n");
```

Comment:
this changes the format, which may break tools that might parse them. Should we backport this?

Reply:
I'm going to move the PR to draft mode as there is consideration for this and other related items.